lib/matrix_check_sum.rb

```diff
@@ -3,8 +3,20 @@
 # of numbers in row i is the same as the sum of numbers in column i for i = 0 to row.length-1
 # If this is the case, return true. Otherwise, return false.
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) - n * m, where n is the number of rows and m is the number of columns
```

Comment:
Time complexity (`O(n^2)`) LGTM, as `n == m` in this problem.

Nit: would `O(n^2)` still hold up if `n != m` though? Why (or why not)?